src/main/java/org/jabref/gui/preferences/xmp/XmpPrivacyTab.java

```diff
@@ -58,7 +65,7 @@ public void initialize() {
         fieldColumn.setReorderable(false);
         fieldColumn.setCellValueFactory(cellData -> BindingsHelper.constantOf(cellData.getValue()));
         new ValueTableCellFactory<Field, Field>()
-                .withText(FieldsUtil::getNameWithType)
+                .withText(item -> FieldsUtil.getNameWithType(item, preferencesService, undoManager))
```

Comment:
Shouldn't the `Injector` be used inside `FieldsUtil` to keep the method signature small?

Reply:
Could be argued. For me the idea was the same with EntryEditor and the RelatedArticlesTab. As afterburner is not an di framework it just provides injection for a presenter in the PresenterFactory. Using the Injector class is not really the intended use and would not be really cleaner than using the Globals class. So I decided for constructor injection here.